portal/src/graphql/portal/LoginMethodConfigurationScreen.tsx

```diff
@@ -794,6 +799,14 @@ function parseOptionalDuration(s: string | undefined) {
   return parseDuration(s);
 }
 
+function parseOptionalDurationInMinutes(s: string | undefined) {
```

Comment:
The name is confusing. This function is not parsing an input "in minutes", it is parsing an duration string into a number in minutes. Maybe we should call it `parseOptionalDurationIntoMinutes(s: string | undefined): number | undefined`.